Memory_Excercises/Excersice10/Source.cpp

```diff
@@ -0,0 +1,12 @@
+#include "ClassA.h"
+#include "ClassB.h"
+#include <iostream>
+
+int main()
+{
+	FClassA ClassA;
+	FClassB ClassB;
+
+	std::cout << "ans 10: the program doesn't work because the Main is calling the header file of both classes so it's making a cycle" << std::endl;
```

Comment:
Perhaps elaborate a bit more... Think of what the exercise tells you to consider, (#include stuff)